ui/src/literature/components/BookSeriesInfo.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+
+class BookSeriesInfo extends Component {
```

Comment:
Maybe `BookChaper` is a better name here

Reply:
However it's only used in one places I would just move this to a function `renderItem` function in the list component file.

Reply:
:+1: